src/config/networks.ts

```diff
@@ -1,27 +1,48 @@
+import { z } from 'zod';
+
 import { entries } from 'utils/type-utils';
 
 import { INFURA_PROJECT_ID } from './constants';
 
-import {
-  IKnownTokenData,
-  INetwork,
-  IToken,
-  KnownContract,
-  KnownToken,
-  NetworkId,
-} from 'types';
+import { IKnownTokenData, INetwork, IToken } from 'types';
+
+// TODO: This are coupled in a few places, make sure that they all
+// stay compatible.
```

Comment:
This could all be cleaned up, I just wanted to make sure the form was compatible with this code and put them in the same place.